fpdf/fpdf.py

```diff
@@ -572,6 +569,64 @@ def set_display_mode(self, zoom, layout="continuous"):
             raise FPDFException(f"Incorrect zoom display mode: {zoom}")
         self.page_layout = LAYOUT_ALIASES.get(layout, layout)
 
+    # Disabling this check - importing outside toplevel to check module is present
+    # pylint: disable=import-outside-toplevel, unused-import
+    def set_text_shaping(
+        self,
+        use_shaping_engine: bool = False,
```

Comment:
Wouldn't it make sense for the default value to be `True`?

I would expect `pdf.set_text_shaping()` to **enable** text shaping, not disable it!